share/functions/__fish_config_interactive.fish

```diff
@@ -299,26 +299,6 @@ function __fish_config_interactive -d "Initializations that should be performed
         set -g fish_pager_color_description yellow
         set -g fish_pager_color_progress cyan
 
```

Comment:
Not exactly. It does not mask off certain colors specifically for `TERM=linux` (or `TERM=vt100`), but the goal is it will work on a VT by not throwing escapes at it that are problematic.